inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -237,8 +237,13 @@ bool layout_optimizer::can_fuse_reorder_to_prev(program_node& prev, program_node
         return true;
 
     if (prev.is_type<permute>()) {
+        auto& permute_order = prev.as<permute>().get_primitive()->permute_order;
+        if (permute_order[1] == 2 && permute_order[2] == 1) {
+            return false;
```

Comment:
As I talked offline, the target case to prevent reorder fusing is 
- fsv format permute => bfyx format reorder 
- permute_order[1] == 2
- "not rotating case" // rotating cases are good to fuse reorder becuase it is  mapped to tiled  kernel